ddtrace/tracer.py

```diff
@@ -230,8 +230,11 @@ def __init__(
         # globally set tags
         self._tags = config.tags.copy()
 
+        # collection of services seen, used for runtime metrics tags
         # a buffer for service info so we don't perpetually send the same things
         self._services = set()  # type: Set[str]
+        if config.service:
```

Comment:
should we add this under line 565 too for after we fork? Or would that be setting the service to config.service which we wouldn't want to assume should happen?

Reply:
yes, good catch!